packages/core/src/index.ts

```diff
@@ -4,6 +4,7 @@ export type { OfflineStore, OfflineTransportOptions } from './transports/offline
 export type { ServerRuntimeClientOptions } from './server-runtime-client';
 
 export * from './tracing';
+export { createEventEnvelope } from './envelope';
```

Comment:
Are we good to export this @mydea?

This also means we'll have to wait for an SDK release before we can release a new version of the SDK

Reply:
fine by me! 👍